.ci/transifex_upload.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Be verbose, and stop with error as soon there's one
+set -ev
+
+
+pip install virtualenv
+virtualenv ~/env
+source ~/env/bin/activate
+pip install transifex-client sphinx-intl
+pip install ".[docs,testing]"
+
+# Create the main database
```

Comment:
Is this is necessary? In principle [it should not be necessary to have a DB for compiling the documentation](https://github.com/aiidateam/aiida_core/blob/6cd8c6ef4ce01ccf864b0c3a60d7d4d7cb0e9d44/docs/source/conf.py#L28)
If there are problems you could also play with setting the `READTHEDOCS` environment variable to `TRUE`, see [here](https://github.com/aiidateam/aiida_core/blob/6cd8c6ef4ce01ccf864b0c3a60d7d4d7cb0e9d44/docs/source/conf.py#L243) and following lines.